app/src/main/java/net/programmierecke/radiodroid2/data/DataRadioStation.java

```diff
@@ -190,4 +207,55 @@ public JSONObject toJson(){
 
 		return null;
 	}
+
+	public interface ShortcutReadyListener {
+		void onShortcutReadyListener(ShortcutInfo shortcutInfo);
+	}
+
+	class RadioIconTarget implements Target {
+		DataRadioStation station;
+		Context ctx;
+		ShortcutReadyListener cb;
+
+		RadioIconTarget(Context ctx, DataRadioStation station, ShortcutReadyListener cb) {
+			super();
+			this.ctx = ctx;
+			this.station = station;
+			this.cb = cb;
+		}
+
+		@Override
+		public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
+			if (Build.VERSION.SDK_INT >= 25) {
+				Intent playByUUUIDintent = new Intent(MediaSessionCallback.ACTION_PLAY_STATION_BY_UUID, null, ctx, ActivityMain.class)
```

Comment:
That's one too many `U`s

Reply:
Thanks, I was already thinking that I need a larger monitor :)